src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.Scanner;
+import java.util.ArrayList; 
+import java.text.ParseException;
+
 public class Duke {
-    public static void main(String[] args) {
+    void logo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+    static void greeting(){
+        String output = "    ____________________________________________________________\n" +
```

Comment:
What i think you can do is to put those formatting lines as a function by itself instead. So everything you have a new message, you can just call the function instead of copy and pasting. 

Also in the event you want to change the format, it will make things a little easier